server/channels/app/imports/import_validators.go

```diff
@@ -312,6 +312,32 @@ func ValidateUserImportData(data *UserImportData) *model.AppError {
 	return nil
 }
 
+func ValidateBotImportData(data *BotImportData) *model.AppError {
```

Comment:
Can we also include this in mmctl/commands/import/validate.go? This will allow us to use `mmctl import validate` for bots.

Reply:
Good call, totally missed that.